src/switch.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * @license
+ * Copyright 2022 MIT
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/**
+ * @fileoverview Multiple selection svg switch.
+ */
+
+import {WorkspaceMultiSelect} from '../src/index';
+import * as Blockly from 'blockly/core';
+
+/**
+ * Icon path of the multi select controls when enabled.
+ * @type {string}
+ * @const
+ */
+const ENABLED_IMG = 'media/select.svg';
+
+/**
+ * Icon path of the multi select controls when disabled.
+ * @type {string}
+ * @const
+ */
+const UNENABLED_IMG = 'media/unselect.svg';
+
+/**
+ * Width of the multi select controls.
+ * @type {number}
+ * @const
+ */
+const WIDTH = 32;
+
+/**
+ * Height of the multi select control.
+ * @type {number}
+ * @const
+ */
+const HEIGHT = 32;
+
+/**
+ * Distance between multi select controls and bottom or top edge of workspace.
+ * @type {number}
+ * @const
+ */
+const MARGIN_VERTICAL = 20;
+
+/**
+ * Distance between multi select controls and right or left edge of workspace.
+ * @type {number}
+ */
+const MARGIN_HORIZONTAL = 20;
+
+/**
+ * Class for a multi select controls.
+ * @implements {Blockly.IPositionable}
+ */
+export class MultiSelectControls {
+  /**
+   * @param {!Blockly.WorkspaceSvg} workspace The workspace to sit in.
+   * @param {!WorkspaceMultiSelect} plugin The plugin to sit in.
+   */
+  constructor(workspace, plugin) {
+    /**
+     * @type {!Blockly.WorkspaceSvg} The workspace to sit in.
+     * @private
+     */
+    this.workspace_ = workspace;
+
+    /**
+     * @type {!WorkspaceMultiSelect} The plugin to sit in.
+     * @private
+     */
+    this.plugin_ = plugin;
+
+    /**
+     * State of the multi select controls.
+     * @type {boolean}
+     */
+    this.enable = false;
```

Comment:
```suggestion
    this.enabled = false;
```

`enable` is a verb, which incorrectly implies this is a function. `enabled` is an adjective, which implies this is a property.